skins/cat17/src/scripts/donationForm.js

```diff
@@ -333,18 +333,23 @@ $( function () {
 		initData.data( 'initial-validation-result' )
 	) );
 
-	var $introBanner = $('.introduction.banner');
-  var $introDefault = $('.introduction.default');
-
-  // @todo Check if this are all conditions that would be considered "successful deeplink", warrant the special header
-  if (initSetup.amount && initSetup.paymentIntervalInMonths && initSetup.paymentType) {
-    $introBanner.removeClass('hidden');
-    $introDefault.addClass('hidden');
-  }
-
 	// Non-state-changing event behavior
 
 	var scroller = WMDE.Scrolling.createAnimatedScroller( $( '.wrap-header, .state-bar' ) );
+
+	// Scroll to first required element that needs to be filled
+	var currentState = store.getState();
+	if ( currentState.validity.paymentData ) {
```

Comment:
This will probably be too simple after #1085. We will see - it's good to see the prototypical code replaced.